Source/WebGPU/WebGPU/USDModel.swift

```diff
@@ -34,6 +34,12 @@ import _USDStageKit_SwiftUI
 import ShaderGraph
 #endif
 
+// FIXME: rdar://166014064 - this shouldn't be necessary yet it seems to be
+extension _USDStageKit_SwiftUI._Proto_MeshDataUpdate_v1 {
```

Comment:
I’d like to be confident that this is indeed a bug in the compiler/Swift and not just a webkit configuration issue first before landing this

Reply:
Seems to be a compiler bug according to the Swift team

Reply:
@rr-codes @adetaylor are we ok merging this since it seems to be a Swift compiler issue?